lib/livebook_web/plugs/auth_plug.ex

```diff
@@ -24,7 +24,9 @@ defmodule LivebookWeb.AuthPlug do
   Stores in the session the secret for the given mode.
   """
   def store(conn, mode, value) do
-    put_session(conn, key(conn.port, mode), hash(value))
+    conn
+    |> put_session(key(conn.port, mode), hash(value))
```

Comment:
```suggestion
    |> put_session(key(conn.port, mode), hash(value))
    # Renew the session to avoid session fixation attacks
    # and to force the cookie to be stored in CSRF errors
```